create-version-file.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+PACKAGE_VERSION=$(node -p "require('./package.json').version")
+echo "{\"version\":\"$PACKAGE_VERSION\"}" > ./dist/apps/admin-gui/assets/config/version.json
+echo "{\"version\":\"$PACKAGE_VERSION\"}" > ./apps/admin-gui/src/assets/config/version.json
```

Comment:
On the app release, you don't want to write a change to the versioned code. This file shouldn't be edited here; it should be done manually or somewhere else.